packages/ui-components/src/header/Header.links.ts

```diff
@@ -0,0 +1,101 @@
+export type MenuItem = {
+  link: string
+  label: string
+  mini?: boolean
+  childs?: MenuItem[]
+}
+
+const navigation: MenuItem[] = [
+  {
+    link: 'https://globalfishingwatch.org/topics/',
```

Comment:
Wouldn't be better to use relative urls for the links so that when this is deployed in any instance is navigable? WDYT @fertandil87 @j8seangel ?

Reply:
I don't think so in this case because some of them could be out of the base url and that url should be permanent

Reply:
Makes sense 👍